app/controllers/users/verify_personal_key_controller.rb

```diff
@@ -12,15 +12,31 @@ def new
         user: current_user,
         personal_key: '',
       )
+
+      if Throttler::IsThrottled.call(current_user.id, :verify_personal_key)
+        render :throttled
```

Comment:
Does it make sense to write something to the analytics hash when we're throttled?

Reply:
Yeah, something should be logged. https://github.com/18F/identity-idp/pull/4803 created new events. Does it make sense to keep that pattern here too (and for an upcoming PR on rate limiting GPO attempts)?

Reply:
See also: LG-4404